zipkin-server/src/test/java/zipkin2/server/internal/Access.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.server.internal;
+
+import zipkin2.collector.Collector;
+
+/** opens package access for testing */
+public final class Access {
+  public static Collector collector(ZipkinHttpCollector httpCollector) {
+    return httpCollector.collector;
+  }
+}
```

Comment:
Interesting pattern. I think I like this more than marking something "VisibleForTesting"